docs/docs/command-line-interface.mdx

```diff
@@ -17,11 +17,12 @@ abstract: The command line interface (CLI) gives you easy-to-remember commands f
 |`rasa shell`            |Loads your trained model and lets you talk to your assistant on the command line.                                                         |
 |`rasa run`              |Starts a server with your trained model.                                                                                                  |
 |`rasa run actions`      |Starts an action server using the Rasa SDK.                                                                                               |
-|`rasa visualize`        |Generates a visual representation of your stories.                                                                                                                                                            |
+|`rasa visualize`        |Generates a visual representation of your stories.                                                                                        |
 |`rasa test`             |Tests a trained Rasa model on any files starting with `test_`.                                                                            |
 |`rasa data split nlu`   |Performs a 80/20 split of your NLU training data.                                                                                         |
 |`rasa data convert`     |Converts training data between different formats.                                                                                         |
-|`rasa data validate`    |Checks the domain, NLU and conversation data for inconsistencies.                                                                                             |
+|`rasa data validate`    |Checks the domain, NLU and conversation data for inconsistencies.                                                                         |
```

Comment:
Can you check why these two show different? It could be just a new line char which is fine but just so that the rendered version doesn't look different

Reply:
Its whitespace changes only according to GitHub - I might have added a space to make then table end pipes align.